tests/libs/ckb-std-wrapper/Cargo.toml

```diff
@@ -7,5 +7,5 @@ edition = "2021"
 native-simulator = ["ckb-std/native-simulator"]
 
 [dependencies]
-ckb-std = { git = "https://github.com/joii2020/ckb-std.git", rev = "a4dfebc" }
+ckb-std = { git = "https://github.com/nervosnetwork/ckb-std.git", rev = "ef251e2" }
```

Comment:
use official version not git dep.

Reply:
ckb-std 0.16.1 is no simulator spawn code.